src/sinks/elasticsearch.rs

```diff
@@ -571,6 +570,7 @@ mod tests {
 mod integration_tests {
     use super::*;
     use crate::{
+        dns::Resolver,
```

Comment:
Is this needed?

Reply:
Yes, we are using it here
https://github.com/timberio/vector/blob/5b610a312eaf67f05fc546d3f66ede679709ad69/src/sinks/elasticsearch.rs#L810
Before it was imported through the above `use super::*;` from which we removed it.